turbinia/workers/hindsight.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Task for executing hindsight."""
+
+from __future__ import unicode_literals
+
+import os
+
+from turbinia.evidence import TextFile
+from turbinia.workers import TurbiniaTask
+
+
+class HindsightTask(TurbiniaTask):
+  """Task to execute hindsight."""
+
+  def __init__(self):
+    super(HindsightTask, self).__init__()
+
+  def run(self, evidence, result):
+    """Task to execute hindsight."""
+
+    # Create the new Evidence object that will be generated by this Task.
+    output_evidence = TextFile()
+
+    # Create a path that we can write the new file to.
+    output_file_path = os.path.join(
+        self.output_dir, '{0:s}'.format(self.id))
+    output_evidence.local_path = output_file_path
+
+    # Create a path for the log file
+    hindsight_log = os.path.join(self.output_dir, '{0:s}.log'.format(self.id))
+
+    cmd = ('hindsight.py -i {0:s} --format={1:s} --browser_type={2:s} '
```

Comment:
We should also add `pyhindsight` into the setup/requirements files as a worker dependency.

Reply:
Thanks for your feedback, Aaron! Will resolve the issues as soon as I get the chance.